### PR TITLE
Require LLVM 7 for centos 7 and rhel 7

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR

### CODE REVIEW

File: default_build_settings.cmake [hunk 1]
```diff
@@ -9,9 +9,9 @@ endfunction()
 
 function(setup_vespa_default_build_settings_rhel_7_7)
   message("-- Setting up default build settings for rhel 7.7")
-  set(DEFAULT_EXTRA_LINK_DIRECTORY "${VESPA_DEPS}/lib64" "/usr/lib64/llvm5.0/lib" PARENT_SCOPE)
-  set(DEFAULT_EXTRA_INCLUDE_DIRECTORY "${VESPA_DEPS}/include" "/usr/include/llvm5.0" "/usr/include/openblas" PARENT_SCOPE)
-  set(DEFAULT_VESPA_LLVM_VERSION "5.0" PARENT_SCOPE)
+  set(DEFAULT_EXTRA_LINK_DIRECTORY "${VESPA_DEPS}/lib64" "/usr/lib64/llvm7.0/lib" PARENT_SCOPE)
+  set(DEFAULT_EXTRA_INCLUDE_DIRECTORY "${VESPA_DEPS}/include" "/usr/include/llvm7.0" "/usr/include/openblas" PARENT_SCOPE)
+  set(DEFAULT_VESPA_LLVM_VERSION "7.0" PARENT_SCOPE)
```
Comment (toregge) [inline default_build_settings.cmake:14]: Use"7", not "7.0" here.

File: dist/vespa.spec [hunk 2]
```diff
@@ -154,14 +154,14 @@ Requires: perf
...
-Requires: llvm5.0
+Requires: llvm7.0
 Requires: vespa-openssl >= 1.1.1c-1
 Requires: vespa-icu >= 65.1.0-1
 Requires: vespa-protobuf >= 3.7.0-4
 Requires: vespa-telegraf >= 1.1.1-1
-%define _vespa_llvm_version 5.0
-%define _extra_link_directory /usr/lib64/llvm5.0/lib;%{_vespa_deps_prefix}/lib64
-%define _extra_include_directory /usr/include/llvm5.0;%{_vespa_deps_prefix}/include;/usr/include/openblas
+%define _vespa_llvm_version 7.0
```
Comment (toregge) [inline dist/vespa.spec:162]: Use 7, not 7.0 here

**Review by toregge — CHANGES_REQUESTED** [review]
You also need to update bootstrap-cmake.sh accordingly.

File: default_build_settings.cmake (hunk 1 above)
Comment (baldersheim) [reply default_build_settings.cmake]: Fixed

File: dist/vespa.spec (hunk 2 above)
Comment (baldersheim) [reply dist/vespa.spec]: Fixed

**Review by toregge — APPROVED** [review]

**Pull request merged**